### DISCUSSION TITLE
Refresh UML Diagram changes

### DISCUSSION THREAD

**Discussion by Alan-Graton** [AbianS/prisma-generate-uml#19] [Ideas]
First of all: loving the last update you made #13, the diagrams are a lot more flexible ^-^

I think we're missing a refresh button, since after every `prisma.schema` change we need to close the extension tab and open it again in order to see the changes.

Not a BIG deal, but would make quite a difference to have it ^-^

**Comment by AbianS** [comment]
Thank you so much for the kind words and feedback! 😊 Instead of adding a refresh button, I decided to make the diagrams update automatically whenever changes are saved in the Prisma schema file. Hope this makes things even smoother for you! 🚀 #20

**Reply by Alan-Graton** [reply]
AWESOME dude! Thanks for the quick reply and for the the hard work ^-^
